src/celesto/computer.py

```
@@ -257,7 +257,20 @@ def recv_loop():
+                try:
+                    with _get_client(api_key) as check_client:
+                        info = check_client.computers.get(resolved_id)
+                        comp_status = info.get("status")
+                        if comp_status == "error":
+                            last_err = info.get("last_error", "unknown")
+                            console.print(f"[red]Computer error: {last_err}[/red]")
+                            console.print("[dim]Create a new computer with: celesto computer create[/dim]")
+                        elif comp_status == "running":
+                            console.print(f"[yellow]Computer is still running. Reconnect with: celesto computer ssh {computer_id}[/yellow]")
+                except Exception:
+                    pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)